modules/tinymce/src/plugins/table/main/ts/api/Events.ts

```diff
@@ -32,16 +33,21 @@ const fireTableSelectionClear = (editor: Editor) => {
   editor.fire('TableSelectionClear');
 };
 
-const fireTableModified = (editor: Editor, table: HTMLTableElement, data?: { structure: boolean; style: boolean }) => {
+const fireTableModified = (editor: Editor, table: HTMLTableElement, data: TableEventData) => {
   editor.fire('TableModified', { ...data, table });
 };
 
+const styleModified = { structure: false, style: true };
+const structureModified = { structure: true, style: false };
```

Comment:
Can we add the `TableEventData` type here to ensure the types are correct?
```suggestion
const styleModified: TableEventData = { structure: false, style: true };
const structureModified: TableEventData = { structure: true, style: false };
```